GVRf/Framework/src/org/gearvrf/GVRViewManager.java

```diff
@@ -462,6 +463,11 @@ void onDrawEyeView(int eye, float fovDegrees) {
         if (!(mSensoredScene == null || !mMainScene.equals(mSensoredScene))) {
             GVRCameraRig mainCameraRig = mMainScene.getMainCameraRig();
 
+            if(eye == 0) {
+                GVRPerspectiveCamera centerCamera = mainCameraRig.getCenterCamera();
+                cull(mMainScene.getNative(), centerCamera.getNative(), mRenderBundle.getMaterialShaderManager().getNative());
+            }
+
```

Comment:
I think it would be better to use a flag to indicate if cull was already calculated. In the past I saw several places in the code where the left eye was not called before the right eye, so we can't assume it.

In general. I would prefer to use LEFT_EYE, RIGHT_EYE, BOTH_EYES enums instead of 0,1
There is already an enum definition in GVRConsole which we have to move to somewhere else and use everywhere.